examples/notebooks/05_market_comparison.ipynb

```diff
@@ -0,0 +1,363 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "4JeBorbE6FYr"
+   },
+   "source": [
+    "# 5. market comparison tutorial\n",
+    "\n",
+    "This tutorial describes a showcase [from the scientific paper \"Market Abstraction of Energy Markets and Policies - Application in an Agent-Based Modeling Toolbox\"](https://link.springer.com/chapter/10.1007/978-3-031-48652-4_10).\n",
```

Comment:
Ideally I would expect people to take a look at the work, but we cannot expect that from everyone? Could you please provide a small theoretical part here, maybe directcly from the paper, introducing the research question and exactly what markets are investigated here?

Reply:
I would say the necessity for this depends on the aim of tutorial. Similar to the new name that you @nick-harder suggested. Is it to show the efects of long-term markets and hence get a different angle on the paper results or to showcase how markets can be configured? 

If it is the later, I would not include any theory besides a description of the markets and how this description is translated to the market-config.

Reply:
we could have both. What if we asked the following RQ? Here is the questions, and we will investigate these market configs. How do we put this into ASSUME? and thus the tutorial.